02-peer-protocol.md

```diff
@@ -7,12 +7,29 @@ operation, and closing.
 
   * [Channel](#channel)
     * [Definition of `channel_id`](#definition-of-channel_id)
-    * [Channel Establishment](#channel-establishment)
+    * [Interactive Transaction Construction](#interactive-transaction-construction)
+      * [Set-Up and Vocabulary](#set-up-and-vocabulary)
+      * [Fee Responsibility](#fee-responsibility)
+      * [Overview](#overview)
+      * [The `tx_add_input` Message](#the-tx_add_input-message)
+      * [The `tx_add_output` Message](#the-tx_add_output-message)
+      * [The `tx_remove_input` and `tx_remote_output` Messages](#the-tx_remove_input-and-tx_remove_output-messages)
+      * [The `tx_complete` Message](#the-tx_complete-message)
+      * [The `tx_signatures` Message](#the-tx_signatures-message)
```

Comment:
It looks like the links for `tx_init_rbf` and `tx_ack_rbf` were lost in the rebase:

```suggestion
      * [The `tx_signatures` Message](#the-tx_signatures-message)
      * [The `tx_init_rbf` Message](#the-tx_init_rbf-message)
      * [The `tx_ack_rbf` Message](#the-tx_ack_rbf-message)
```

Reply:
Oops. Will fix...